website/settings/defaults.py

```diff
@@ -122,7 +122,12 @@ def parent_dir(path):
 
 USE_EMAIL = True
 FROM_EMAIL = 'openscienceframework-noreply@osf.io'
-SUPPORT_EMAIL = 'support@osf.io'
+
+if not DEV_MODE and not DEBUG_MODE:
```

Comment:
One reason this ticket surfaced was that other people have been hosting their own versions of the OSF (albeit locally for the most part). I think the preferred route is to not configure a support email automatically at all and to have devops add it to their local config.